app/forms/signups/create_form.rb

```diff
@@ -0,0 +1,55 @@
+module Signups
+  class CreateForm
+    include ActiveModel::Model
+
+    EN_US_LOCALE = 'en-US'.freeze
+
+    attr_accessor :account_id, :email, :installation_id, :structured_address
+
+    validates :account_id, presence: true
+    validates :installation_id, presence: true
+    validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+    validate :email_uniqueness
+
+    def submit
+      return unless valid?
```

Comment:
Na minha visão fica mais claro:
```suggestion
      return if invalid?
```
O que acha?

Reply:
Verdadeiro!

Como já abri vários PRs depois desse, corrigi na branch mais recente pra evitar conflitos. :praise:
cfb85341087628d019f7fc4be51dc0da0ca0fd25